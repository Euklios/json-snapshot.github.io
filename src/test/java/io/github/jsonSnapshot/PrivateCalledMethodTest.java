package io.github.jsonSnapshot;

import static io.github.jsonSnapshot.SnapshotMatcher.expect;

import org.junit.jupiter.api.AfterAll;
import org.junit.jupiter.api.BeforeAll;
import org.junit.jupiter.api.Test;

class PrivateCalledMethodTest {

  @BeforeAll
  static void beforeAll() {
    SnapshotMatcher.start();
  }

  @AfterAll
  static void afterAll() {
    SnapshotMatcher.validateSnapshots();
  }

  @Test
  void testName() {
    testBasedOnArgs("testContent");
  }

  private void testBasedOnArgs(String arg) {
    expect(arg).toMatchSnapshot();
  }
}
