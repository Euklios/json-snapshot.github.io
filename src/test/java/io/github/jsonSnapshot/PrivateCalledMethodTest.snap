io.github.jsonSnapshot.PrivateCalledMethodTest.testName=[
  "testContent"
]